knox/auth.py

```diff
@@ -34,7 +34,9 @@ class TokenAuthentication(BaseAuthentication):
     - `request.user` will be a django `User` instance
     - `request.auth` will be an `AuthToken` instance
     '''
-    model = AuthToken
+
+    def get_model(self):
+        return get_token_model()
```

Comment:
I would prefer to have this declared on the top of the file, in the same fashion the `User` model is declared.
What would be the use case to override this method?